src/settings.qml

```diff
@@ -465,8 +465,12 @@ import Qt.labs.settings 1.0
             // from the version 2.11.41
             property bool fakedevice_treadmill: false
 
-            // from the version 2.11.43
+			// from the version 2.11.43
             property int video_playback_window_s: 12
+
+            // from the version ?
```

Comment:
Need a specific version here.
Also,  "the" is not required in these:

i.e. "From version 2.11.44" is better IMO. I'm just copying the current style.

Reply:
yes you're right :) i will remove "the" and putting the right version when i will merge it